tests/test_meetings.py

```diff
@@ -31,55 +33,72 @@ def test_meetings_landing(self, meetings_page):
         assert meetings_page.nrao_logo.present()
         assert meetings_page.spsp_logo.present()
 
-    def test_filtering(self, meetings_page):
+    def test_filtering(self, meetings_page, driver):
+        search_bar = driver.find_element_by_css_selector('div[data-test-meetings-list-search]')
+        driver.execute_script('arguments[0].scrollIntoView();', search_bar)
+
         default_top_result = meetings_page.top_meeting_link.text
         meetings_page.filter_input.clear()
-        meetings_page.filter_input.send_keys('ea')
+        meetings_page.filter_input.send_keys('f')
+        time.sleep(.5)
         filtered_top_result = meetings_page.top_meeting_link.text
         assert default_top_result != filtered_top_result
 
-    def test_carets(self, meetings_page):
+    def test_carets(self, meetings_page, driver):
+        search_bar = driver.find_element_by_css_selector('div[data-test-meetings-list-search]')
+        driver.execute_script('arguments[0].scrollIntoView();', search_bar)
+
         default_top_result = meetings_page.top_meeting_link.text
-        meetings_page.sort_caret_name_desc.click()
+        meetings_page.sort_caret_name_asc.click()
```

Comment:
Let's triple check that this passes on all envs

Reply:
Update: it does pass on all envs! But we made the choice to sort by desc just to future proof it.